### PR TITLE
Sonarqube migration

### PR DESCRIPTION
**Pull request by laedit**
I was trying to migrate SonarQubeHelper to FAKE 5, but I can't use Fake-netcore.sln: I got this error: Unrecognized Guid format..

So I made it all by hand and I got the following error on build:

    FS0039: D:\prog\github\fake\src\app\Fake.Testing.SonarQube\SonarQube.fs(15,15): The namespace 'Core' is not defined.

Can you help me to finish this?

### CODE REVIEW

**Comment by matthid** [comment]
Oh yeah sure. 
First: I don't even use Fake-netcore.sln myself, it's there just to make building steps in FAKE easier (and faster actually).
Usually what I do is:
 - Copy one of the existing netcore projects and edit the project file by hand.
 - Copy the old implementation files (update project file again if required)
 - Reference the new files in FakeLib (again updating FakeLib.fsproj by hand to properly reference the stuff)
 - Open Fake.sln and go from there. Because in F# you can only reference stuff defines in files ABOVE, this is ALMOST perfect
 - Once stuff compiles in the (Fake.sln) solution the remaining changes to make the netcore project compile are usually straightforward (you basically only need to fix project references or add framework nuget packages if needed)

Thanks for bringing that to my attention, will add that to the docs somewhere.
Let me know if that actually helps.

**Comment by laedit** [comment]
Thanks, I have already did the first three but haven't though of continuing with `Fake.sln`! 😄

It should be all good now.

**Comment by laedit** [comment]
It seems that Travis is struggling during DotnetPack but I don't know why.

**Comment by matthid** [comment]
Yes mono is broken, don't worry about it. (https://github.com/fsprojects/Paket/issues/2393)

File: FAKE.sln [hunk 1]
```diff
@@ -66,7 +66,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "help", "help", "{564BD198-0
 		help\nuget.md = help\nuget.md
 		help\octopusdeploy.md = help\octopusdeploy.md
 		help\slacknotification.md = help\slacknotification.md
-		help\sonarcube.md = help\sonarcube.md
 		help\specifictargets.md = help\specifictargets.md
 		help\teamcity.md = help\teamcity.md
 		help\typescript.md = help\typescript.md
```
Comment (matthid) [inline FAKE.sln]: Ah yes, I should probably update those, as I moved them into a subfolder..

**Comment by matthid** [comment]
Released in alpha010

**Pull request merged**